tests/test_app.py

```diff
@@ -50,4 +50,31 @@ def test_add_sample_labels(self):
                 data=flask.json.dumps(event),
                 content_type='application/json'
             )
-            self.assertEqual(response.status_code, 201)
+            self.assertEqual(response._status_code, 201)
+
+    def test_date_range(self):
```

Comment:
This code was moved to `test_events.py`. It looks like the change is the addition of the last subtest. It should go in that file instead of adding it here.